package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think you might have forgotten to remove this file 😆

Reply:
yes........